econml/inference.py

```diff
@@ -1216,11 +1161,16 @@ def _mixture_ppf(self, alpha, mean, stderr, tol):
         """
         Helper function to get the confidence interval of mixture gaussian distribution
         """
-        done = False
+        # if stderr is zero, ppf will return nans and the loop below would never terminate
+        # so bail out early; note that it might be possible to correct the algorithm for
+        # this scenario, but since scipy's cdf returns nan whenever scale is zero it won't
+        # be clean
+        if (np.any(stderr == 0)):
```

Comment:
This branch is never tested. So we never test population summary with zero stderrs.

Reply:
Added a test.